src/net.cpp

```diff
@@ -1670,12 +1671,12 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
                 LogPrintf("Adding fixed seeds as 60 seconds have passed and addrman is empty for at least one reachable network\n");
             }
 
```

Comment:
nit: maybe remove this line between the if/else?

Reply:
I think I'll remove it in a follow-up #26114 unless I need retouch for another reason.